### PR TITLE
fix: fix backwards compatibility to 1.14

### PR DESCRIPTION
**Pull request by emilioalvap**
- Fix backwards compatibility to 1.14

### CODE REVIEW

File: src/core/runner.ts [hunk 1]
```diff
@@ -131,8 +128,20 @@ export default class Runner implements RunnerInfo {
     this.#hooks[type].push(callback);
   }
 
+  /**
+   * @deprecated Since version 1.17.0. Use _updateMonitor instead.
```
Comment (vigneshshanmugam) [inline src/core/runner.ts:132]:
```suggestion
   * @deprecated Since version 1.17.0. Please do not rely on the internal methods. 
```

Comment (vigneshshanmugam) [reply src/core/runner.ts:132]: WDYT?

**Review by vigneshshanmugam — COMMENTED** [review]
LGTM, but instead of alias, I would very much in favor of asking users to not rely on these methods or callout saying use it with caution.

File: src/core/runner.ts (hunk 1 above)
Comment (emilioalvap) [reply src/core/runner.ts:132]: I'm OK with that, will update.

**Review by vigneshshanmugam — APPROVED** [review]

**Pull request merged**